managed-services/transactional-email/api-cli/send-emails-with-api.mdx

```diff
@@ -0,0 +1,131 @@
+
+
+1. Open a terminal and paste the following code to configure your environment variables. Make sure that you add your own values.
+
+        ```
+        export SCW_ACCESS_KEY="<API access key>"
+        export SCW_SECRET_KEY="<API secret key>"
+        export SCW_PROJECT_ID="<Scaleway Project ID>"
+        ```
+2. Run the following command to retrieve your domain's ID, as you will need it in the next step. The output should return your domain's information.
+        ```
+        curl -X GET \
+        -H "X-Auth-Token: $SCW_SECRET_KEY" \
+        "https://api.scaleway.com/transactional-email/v1alpha1/regions/fr-par/domains"
+        ```
+3. Run the following command to query your domain. Make sure that you replace `<domain-id>` with the ID of your domain.
+        ```
+        curl -X POST "https://api.scaleway.com/transactional-email/v1alpha1/regions/$REGION/domains/<domain-id>/check" \
+            -H "X-Auth-Token: $SCW_SECRET_KEY" \
+            -d "{}"
+
+4. Run the following command to ensure that your domain is verified:
+
+        ```
+        curl -X GET "https://api.scaleway.com/transactional-email/v1alpha1/regions/$REGION/domains/<domain-id>" \
+        -H "X-Auth-Token: $SCW_SECRET_KEY"
+        ```
+
+5. Copy the following template. Make sure that you replace the placeholder information with your own.
+
+        ```
+        cat > mail.json <<EOF
+        {
+            "from": {
+                "name": "Me", # Replace 'Me' with your own name
+                "email": "me@my.domain.example.com"  # Replace 'me@my.domain.example.com' with your email address
+            },
+            "to": [
+                {
+                    "name": "Your recipient", # Replace 'Your recipient' with your recipient's name
+                    "email": "your.recipient@mail.fr" # Replace 'your.recipient@mail.fr' with your recipient's email address
+                }
+            ],
+            "subject": "This is a subject", # Replace with your subject. Subjects must have at least 10 characters
+            "text": "This is a short sentence.", # Replace with the body of your email
+            "project_id": "<Scaleway Project ID>", # Replace '<Scaleway Project ID>' with your Scaleway Project ID
+            "attachments": [
+                {
+                    "name": "file.html", # Replace with the name of the file your want to send
+                    "type": "text/html; charset=\"utf8\"", # Replace with the attachment type you want to send
+                    "content": "PCFET0NUWVBFIGh0bWw+CjxodG1sIGxhbmc9ImVuIj4KICAgIDxoZWFkPgogICAgICAgIDxtZXRhIGNoYXJzZXQ9InV0Zi04Ij4KICAgICAgICA8dGl0bGU+dGl0bGU8L3RpdGxlPgogICAgPC9oZWFkPgogICAgPGJvZHk+CiAgICAgICAgPHA+SGVsbG8gPHNwYW4gc3R5bGU9ImNvbG9yOmJsdWUiPndvcmxkPC9zcGFuPiE8L3A+CiAgICA8L2JvZHk+CjwvaHRtbD4=" # Replace with the content you want to send
+                }
+            ],
```

Comment:
```suggestion
            "project_id": "<Scaleway Project ID>", # Replace '<Scaleway Project ID>' with your Scaleway Project ID
            "text": "This is a short sentence.", # Replace with the body of your email
            "html": "<html><body><p>This is a short sentence.</p></body></html>", # Replace with the content you want to send,
            "attachments": [
		{
			"name": "test.pdf",
			"type": "application/pdf",
			"content": "AAAA==" # Your PDF encoded in Base64
		}
            ],
```

This example of HTML file attachment, not the best one. Add an HTML in attachment is a really special case. You may want to add this HTML example in the "html" field, like this. Instead, a PDF kind be a good example.